### DISCUSSION TITLE
Consider using Praat 6.4

### DISCUSSION THREAD

**Discussion by dan-ya** [YannickJadoul/Parselmouth#117] [General]
Praat has a slightly improved pitch detection algorithm since version 6.4 (released at November 15, 2023). It could be beneficial to use some version of Praat 6.4.*. The consequence of the improvement that they changed the names of pitch detection functions (see, https://www.fon.hum.uva.nl/praat/manual/What_s_new_.html).

**Comment by YannickJadoul** [comment]
Yes, Praat is due a big update, and that's actually on the list to get into 0.5.0: see #107.
It's not been easy finding the time to do so, but I have finally planned to get some work on this done next week.

Regarding the new pitch functions: any suggestions on how the Python API would look? We currently have `to_pitch` (generic), `to_pitch_ac`, `to_pitch_cc`, etc.
I assume they will have to be added as new methods, `to_pitch_filtered_ac` for example, to keep backwards compatibility. But curious to hear any thoughts you might have!

**Reply by dan-ya** [reply]
Thank you for considering to include it! it would be great!

I think it would be fine to have a new method `to_pitch_filtered_ac`, if it works out. Thus, Parselmouth users would know that it is a different method. Although, the new users of Parselmouth, who would come from Praat, could be confused, as in Praat, they now have `to_pitch_filtered_ac` as a baseline for `to_pitch_ac`, while the previous `to_pitch_ac` is now renamed to `to_pitch_ac_raw`.
